docs/superpowers/plans/2026-05-19-rpi4-bcm2709-alignment.md

```
@@ -0,0 +1,1166 @@
+Run: `diff -rq conf/full_raspberrypi_bcm27xx_bcm2712 conf/full_raspberrypi_bcm27xx_bcm2709`
+Expected: output matches the deltas enumerated in the "File Structure" section above. If the diff has *new* deltas (e.g. someone added a file to bcm2712 since this plan was written), stop and re-scope. If the diff is *missing* deltas (someone already aligned partially), that's fine — proceed and the tasks below will idempotently overwrite.
+
+Run: `diff -rq conf/base_raspberrypi_bcm27xx_bcm2712 conf/base_raspberrypi_bcm27xx_bcm2709`
+Expected: similar verification for base profile.
+
+- [ ] **Step 4: Snapshot current bcm2709 .config target lines for restoration reference**
+
+Run: `grep -E '^CONFIG_TARGET_bcm27xx|^CONFIG_TARGET_SUBTARGET|^CONFIG_TARGET_PROFILE|^CONFIG_TARGET_ARCH_PACKAGES|^CONFIG_CPU_TYPE|^CONFIG_ARCH|^CONFIG_aarch|^CONFIG_TARGET_SUFFIX' conf/full_raspberrypi_bcm27xx_bcm2709/.config`
+
+Expected output (these are the lines we MUST preserve through all subsequent edits):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)